.github/workflows/cpu-tests.yml

```diff
@@ -3,8 +3,18 @@ name: CPU tests
 on:
   push:
     branches: [main, wip]
+    paths:
+      - '**/*.py'
```

Comment:
based on your description, shouldn't it be

```
- "!*.md"
- "!**/*.md"
```

Reply:
Sure, that might be better and more concise. Let's try that.